src/main/scripts/src/app/shared/perspectives/collections/post-it/post-it-collections-perspective.component.html

```diff
@@ -23,8 +38,8 @@
 
       <div class="content" [style.color]="collection.color">
         <i class="fa fa-2x {{ collection.icon }}" title="Go to documents"
-           [routerLink]="['..', 'c', collection.code, 'documents']"></i><!--
-      --><input [(ngModel)]="collection.name" (ngModelChange)="updateCollection(collection)"/>
+           [routerLink]="['..', 'c', collection.code, 'documents']"></i>
```

Comment:
The comments are there to remove added space character (https://css-tricks.com/fighting-the-space-between-inline-block-elements/)

Reply:
Oh, sorry. I will give it back..